# This message holds the data of an object detected by vision
int32 label
string labelText
float32 score
float32 ymin
float32 xmin
float32 ymax
float32 xmax
geometry_msgs/Point point3D